package.json

```diff
@@ -182,6 +182,11 @@
 					],
 					"default": "colour",
 					"description": "Specifies the colour theme of the icon displayed on the Git Graph tab."
+				},
+				"git-graph.retainContextWhenHidden": {

```

Comment:
Can this property please be moved to line 158 so that configuration properties are kept in alphabetical order.